Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Substance/o3de_dccsi.sbsprj

```diff
@@ -0,0 +1,391 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<root>
+	<showresourcesinlibrary>true</showresourcesinlibrary>
+	<showgraphsinlibrary>true</showgraphsinlibrary>
+	<settingsinfo>
+		<version>2</version>
+	</settingsinfo>
+	<projectinfo>
+		<label>O3de dccsi</label>
+	</projectinfo>
+	<preferences>
+		<view3d>
+			<statesavefileurl>file:///C:/Users/gallowj/AppData/Local/Adobe/Adobe Substance 3D Designer/user_view3d_settings.sbsscn</statesavefileurl>
```

Comment:
Yeah, so Substance is kind of clever and also kind of terrible sometimes, this is one example.  Thanks for catching this.  IF you press the "apply" button in the preferences AFTER loading a custom config like this, it will do things like expand what were relative paths into abs paths, or write other prefs like this into the current config.  I think really what they should have done is write user data to the user config always (because they have support for a hierarchical and layered config system already.)  So I will have to just check and clean before a commit.